pep-0557.rst

```diff
@@ -186,10 +194,10 @@ signature.  That is, these three uses of ``@dataclass`` are equivalent::
 
 The parameters to ``dataclass`` are:
 
-- ``init``: If true (the default), a ``__init__`` method will be
```

Comment:
I always read these as "a dunder init method", so the original sounds correct to me.

Reply:
Fair enough.